tools/actions/build-checks/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "build": "tsup -d build src/main.ts",
+    "build": "tsup --target node20 -d build src/main.ts",
```

Comment:
do we want to create a trail for why this has been added? Currently there's no explanation in the commit message or the PR description

Reply:
I rewrote the history to make it clear